mteb/tasks/Retrieval/eng/IFIRCdsRetrieval.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class IFIRCds(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="IFIRCds",
+        dataset={
+            "path": "if-ir/cds",
+            "revision": "c406767",
+        },
+        description="BBenchmark IFIR cds subset within instruction following abilities.",
```

Comment:
The description has to be detailed enough for me to get a gist of what the dataset contains.